components/scripts/gradle/04-validate-remote-build-caching-ci-ci.sh

```diff
@@ -137,7 +137,7 @@ parse_build_scan_urls() {
       base_urls+=("${protocol}://${ge_host}${port}")
       build_scan_ids+=("$build_scan_id")
     else
-      die "${url} is not a parsable URL." 4
+      die "${url} is not a parsable URL."
```

Comment:
@jthurne should we return "${INVALID_INPUT}" too here?

Reply:
Yes, you're right. Will fix.

Reply:
✅ fixed